app/components/Riddle.jsx

```diff
@@ -0,0 +1,46 @@
+import React, {Component} from 'react';
+
+import {GridList, GridTile} from 'material-ui/GridList';
```

Comment:
it doesn't look like these material ui imports are being used in this component. (lines 3-15)